### PR TITLE
Feature1

### PR DESCRIPTION
**Pull request by AMRUTHA-BYSANI**
Thank you for making a pull request to ZTM's Animation Nation project. We will review your pull request as soon as we can, but keep in mind we can get very busy during Hacktoberfest.

While you're waiting, please look over this checklist. Feel free to write a description of what you did as well!

## What I did

*Optional: Edit this to tell us what you did!*

## Pull request checklist

- [x] This pull request uses **HTML and CSS** only and provides a `meta.json` file.
- [x] All files in this pull request are in a directory named: `<github_username>-<art_name>` without the <>.
- [x] Your pull request contains `index.html` and `styles.css`.
	- [x] Your html file is named exactly: `index.html`
	- [x] Your css file is named exactly: `styles.css`
- [x] Your pull request your `meta.json` with:
	- [x] `githubHandle`: being your unique github user name,
	- [x] `artName` : name of your animation
- [x] Your submission includes **at least one animation**.
- [x] Your pull request **does not modify any other files** in the repository.

#### Not normally needed
- [x] Check this if you would like to discuss an improvement besides adding animations

**Note: if your pull request makes an improvement besides adding a CSS animation you will need to discuss this with the maintainers. You can help kickstart this process by explaining what your change does and why you are making it in the "What I did" section**

## What to expect

- If your pull request meets the checklist requirements it will be approved and merged by a maintainer
- If your pull request needs some changes. A maintainer will leave a comment requesting changes
- If your pull request significantly violates the criteria in the checklist it will be closed with feedback.

### CODE REVIEW

File: Art/AmruthaBysani-LoadingColoursAnimation/styles.css
Comment (LaurelineP) [inline Art/AmruthaBysani-LoadingColoursAnimation/styles.css:1]: # Unnecessary comments
- [ ] Please remove the comments

**Review by LaurelineP — CHANGES_REQUESTED** [review]
Aloha @AMRUTHA-BYSANI 
Thanks for your contribution 🙌

Before we could merge your PR 
- [ ] Please address the other given feedbacks in your PR
- [ ] Please remain consistent between the folder naming and your handle
The folder must be <github-username>-<artname>:
  - your folderName is capitalising and using camelcase - which is not your github username (= github handle )
 Please adjust it

**Comment by AMRUTHA-BYSANI** [comment]
requested changes are made!!

**Comment by LaurelineP** [comment]
@AMRUTHA-BYSANI 
As you can see:
-  in the above ping
- in your branch name ( see picture ) 
image

Your github handle in AMRUTHA-BYSANI not any other proposition
The modifications you've set did the opposite.
The detailed review was pointing the incorrectness made ( github handle stated as capitalised and camelcased folder name ) 

- [ ] Please rename your folder accordingly
- [ ] Please re-adjust your meta ( which was earlier correct ) accordingly

**Comment by AMRUTHA-BYSANI** [comment]
requested changes are made!!

File: Art/AMRUTHA-BYSANI-LoadingColoursAnimation/meta.json
Comment (LaurelineP) [inline Art/AMRUTHA-BYSANI-LoadingColoursAnimation/meta.json:1]: - Please correct the githubHandle as you did for your folder and explained in previous feedbacks :)

**Review by LaurelineP — CHANGES_REQUESTED** [review]
In your meta, the github handle is not the correct one - use the same as you've adjusted the name of your folder

File: Art/AMRUTHA-BYSANI-LoadingColoursAnimation/meta.json
Comment (LaurelineP) [reply Art/AMRUTHA-BYSANI-LoadingColoursAnimation/meta.json:1]: @AMRUTHA-BYSANI : up

**Comment by LaurelineP** [comment]
@AMRUTHA-BYSANI 
I've fixed the pr for you, however: not having activities for 5+ more days generally is a good indicator there are no development, for maintaining reason next time this will cause a MR to be closed

**Pull request merged**